### ISSUE TITLE
关于增值税发票关系抽取，无法加载模型，一直卡在build_model(config['Architecture'])部分

### ISSUE THREAD

**Issue by explorer009** [PaddlePaddle/PaddleOCR#9164]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment： win10
- 版本号/Version：release-2.6   Paddle：2.4.1.post112
 PaddleOCR：2.6.1.3   问题相关组件/Related components：
- 运行指令/Command Code： python tools/infer_kie_token_ser_re.py -c configs/kie/vi_layoutxlm2/re_vi_layoutxlm_xfund_zh.yml -o Architecture.Backbone.checkpoints=inference/fapiao/re/best_accuracy Global.infer_img=doc/fapiao Global.infer_mode=False -c_ser configs/kie/vi_layoutxlm2/ser_vi_layoutxlm_xfund_zh.yml -o_ser rchitecture.Backbone.checkpoints=inference/fapiao/ser/best_accuracy
-(上述命令是对 增值税发票进行关系抽取的命令)
- 完整报错/Complete Error Message：  无报错信息，程序卡在加载 关系抽取模型的位置

我的完整的 re配置文件如下：
Global:
  use_gpu: True
  epoch_num: &epoch_num 130
  log_smooth_window: 10
  print_batch_step: 10
  save_model_dir: ./output/re_vi_layoutxlm_xfund_zh
  save_epoch_step: 2000
  # evaluation is run every 10 iterations after the 0th iteration
  eval_batch_step: [ 0, 19 ]
  cal_metric_during_train: False
  save_inference_dir:
  use_visualdl: False
  seed: 2022
  infer_img: ppstructure/docs/kie/input/zh_val_21.jpg
  save_res_path: ./output/re/xfund_zh/with_gt
  kie_rec_model_dir: 
  kie_det_model_dir:

Architecture:
  model_type: kie
  algorithm: &algorithm "LayoutXLM"
  Transform:
  Backbone:
    name: LayoutXLMForRe
    pretrained: True
    mode: vi
    checkpoints:

Loss:
  name: LossFromOutput
  key: loss
  reduction: mean

Optimizer:
  name: AdamW
  beta1: 0.9
  beta2: 0.999
  clip_norm: 10
  lr:
    learning_rate: 0.00005
    warmup_epoch: 10
  regularizer:
    name: L2
    factor: 0.00000
    
PostProcess:
  name: VQAReTokenLayoutLMPostProcess

Metric:
  name: VQAReTokenMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/zzsfp/imgs
    label_file_list: 
      - train_data/zzsfp/train.json
    ratio_list: [ 1.0 ]
    transforms:
      - DecodeImage: # load image
          img_mode: RGB
          channel_first: False
      - VQATokenLabelEncode: # Class handling label
          contains_re: True
          algorithm: *algorithm
          class_path: &class_path train_data/zzsfp/class_list.txt
          use_textline_bbox_info: &use_textline_bbox_info True
          order_method: &order_method "tb-yx"
      - VQATokenPad:
          max_seq_len: &max_seq_len 512
          return_attention_mask: True
      - VQAReTokenRelation:
      - VQAReTokenChunk:
          max_seq_len: *max_seq_len
      - TensorizeEntitiesRelations:
      - Resize:
          size: [224,224]
      - NormalizeImage:
          scale: 1
          mean: [ 123.675, 116.28, 103.53 ]
          std: [ 58.395, 57.12, 57.375 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'input_ids', 'bbox','attention_mask', 'token_type_ids', 'entities', 'relations'] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 2
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/zzsfp/imgs
    label_file_list:
      - train_data/zzsfp/val.json
    transforms:
      - DecodeImage: # load image
          img_mode: RGB
          channel_first: False
      - VQATokenLabelEncode: # Class handling label
          contains_re: True
          algorithm: *algorithm
          class_path: *class_path
          use_textline_bbox_info: *use_textline_bbox_info
          order_method: *order_method
      - VQATokenPad:
          max_seq_len: *max_seq_len
          return_attention_mask: True
      - VQAReTokenRelation:
      - VQAReTokenChunk:
          max_seq_len: *max_seq_len
      - TensorizeEntitiesRelations:
      - Resize:
          size: [224,224]
      - NormalizeImage:
          scale: 1
          mean: [ 123.675, 116.28, 103.53 ]
          std: [ 58.395, 57.12, 57.375 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'entities', 'relations'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 8
    num_workers: 8

**Comment by explorer009** [comment]
[2023-02-21 10:10:19,231] [    INFO] - Already cached C:\Users\SEVEN\.paddlenlp\models\layoutxlm-base-uncased\layoutxlm-base-uncased\sentencepiece.bpe.model
[2023-02-21 10:10:19,738] [    INFO] - tokenizer config file saved in C:\Users\SEVEN\.paddlenlp\models\layoutxlm-base-uncased\layoutxlm-base-uncased tokenizer_config.json
[2023-02-21 10:10:19,741] [    INFO] - Special tokens file saved in C:\Users\SEVEN\.paddlenlp\models\layoutxlm-base-uncased\layoutxlm-base-uncased\special_tokens_map.json

以上是运行程序之后的控制台信息，再无其他，就卡住了

**Comment by andyjpaddle** [comment]
跑其他模型正常吗

**Comment by explorer009** [comment]
其他的是正常的，做增值税发票识别的第三步----语义实体识别 （Semantic Entity Recognition）是正常的，可以识别实体。但是将语义实体识别 和  关系抽取联合 就出不来了。

**Comment by explorer009** [comment]
> 跑其他模型正常吗

其他的是正常的，做增值税发票识别的第三步----语义实体识别 （Semantic Entity Recognition）是正常的，可以识别实体。但是将语义实体识别 和 关系抽取联合 就出不来了。

**Comment by andyjpaddle** [comment]
升级一下paddlenlp试试

**Comment by explorer009** [comment]
> 升级一下paddlenlp试试
好
